rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/util/TbMsgSource.java

```diff
@@ -13,9 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.thingsboard.rule.engine.metadata;
+package org.thingsboard.rule.engine.util;
 
-public enum FetchTo {
+public enum TbMsgSource {
```

Comment:
TbMsgItem? TbMsgSource is not a clear naming. Even FetchTo is better.

Reply:
It was renamed and moved to the util package to reuse as fetchFrom, copyFrom, deleteFrom, renameIn properties in other nodes. I will think about better naming and will back to you.

Reply:
Given that the property refers specifically to DATA and METADATA fields, here are some alternative suggestions that still maintain the idea of "source" but provide more context:

TbMsgPayloadSource - Emphasizes that it represents the source of the payload data within the message.
TbMsgContentSource - Focuses on the source of the content (data/metadata) within the message.
TbMsgFieldSource - Indicates the source of the fields (data/metadata) within the message.
TbMsgDataOrigin - Specifies the origin of the data within the message.
TbMsgPayloadOrigin - Emphasizes the origin of the payload (data/metadata) within the message.